### PR TITLE
Adding option for parseInModule since it takes so long for some definitions.

### PR DESCRIPTION
**Pull request by radumereuta**
@dwightguth please review.
I also found the 'loud' option which is not being used anymore, so I removed it.

### CODE REVIEW

File: kernel/src/main/java/org/kframework/kompile/KompileOptions.java [hunk 1]
```diff
@@ -129,8 +129,9 @@ public String syntaxModule() {
         @Parameter(names="--no-prelude", description="Do not include anything automatically.")
         public boolean noPrelude = false;
 
-        @Parameter(names="--loud", description="Prints 'Done' at the end if all is ok.")
-        public boolean loud = false;
+        // TODO: (RaduM) this might be temporary, since I want to generate the parsers by need if the definition is loaded
+        @Parameter(names="--parseInModule", description="Generates all program parsers for each module.")
```
Comment (dwightguth) [inline kernel/src/main/java/org/kframework/kompile/KompileOptions.java]: call this --java-parser and give it a description accordingly.

**Comment by radumereuta** [comment]
Done.

File: kernel/src/main/java/org/kframework/kompile/KompileOptions.java [hunk 2]
```diff
@@ -129,8 +129,9 @@ public String syntaxModule() {
         @Parameter(names="--no-prelude", description="Do not include anything automatically.")
         public boolean noPrelude = false;
 
-        @Parameter(names="--loud", description="Prints 'Done' at the end if all is ok.")
-        public boolean loud = false;
+        // TODO: (RaduM) this might be temporary, since I want to generate the parsers by need if the definition is loaded
+        @Parameter(names="--java-parser", description="Generate tables for the new java parser.")
+        public boolean parseInModule = false;
```
Comment (dwightguth) [inline kernel/src/main/java/org/kframework/kompile/KompileOptions.java]: rename the field too?

**Comment by dwightguth** [comment]
I believe you will need to modify the config for some of the tests in the test suite.

**Comment by radumereuta** [comment]
Somehow I forgot to rename the field. Should be fixed now.
I have no idea why the test failed the first time. Could you have a look?

**Comment by radumereuta** [comment]
Everything passed.

**Pull request merged**